streamz/dataframe/core.py

```diff
@@ -283,7 +289,13 @@ def concat(tup, columns=None):
                                     for k, v in args[0].items()})
             DataFrame.__init__(self, stream, example)
         else:
-            return super(DataFrame, self).__init__(*args, **kwargs)
+            example = kwargs.get('example', None) if "example" in kwargs else args[1]
+            assert example is not None
```

Comment:
Can we raise an AttributeError here instead of asserting?

Reply:
Handled this in the latest commit